pkg/solana/txm/txm_internal_test.go

```diff
@@ -680,6 +680,8 @@ func TestTxm_Enqueue(t *testing.T) {
 	lggr := logger.Test(t)
 	cfg := config.NewDefault()
 	mc := mocks.NewReaderWriter(t)
+	mc.On("GetLatestBlock", mock.Anything).Return(&rpc.GetBlockResult{}, nil).Maybe()
```

Comment:
this will be needed when we change the configs. This was detected when running Soak Tests

Reply:
We'll actually keep the default estimator as `fixed` to avoid any changes in the existing behavior. Would we really need these?

Reply:
We will need them to be able to set the `FeeEstimatorMode = "blockhistory"` without tests failing: 
- When `BlockHistorySize = 1`, we [need](https://github.com/smartcontractkit/chainlink-solana/blob/213f7bc4d55149410327edc95c6b421d0458775f/pkg/solana/fees/block_history.go#L124) `GetLatestBlock` mock
- When `BlockHistorySize > 1`, we [need](https://github.com/smartcontractkit/chainlink-solana/blob/213f7bc4d55149410327edc95c6b421d0458775f/pkg/solana/fees/block_history.go#L167) `SlotHeight` mock

Reply:
Isn't this test just using the default fixed price estimator though so don't think these mocks will be used. We could add these if we were explicitly adding tests for BHE but don't think we are?

Reply:
ohh now I get it. Yes, you are right